src/Dropdown/Dropdown.js

```diff
@@ -19,23 +19,60 @@ export default class Dropdown extends Util.mixin(BindMixin) {
   constructor() {
     super();
     this.state = {
+      menuDirection: 'down',
+      menuHeight: null,
       isOpen: false,
-      selectedID: null
+      selectedID: null,
+      knowMenuHeight: false
```

Comment:
I see, so if we re-render the dropdown while it's already open it'll be smart enough to toggle where it's placed? Do you think that's the best UX?